doc/api_assets/style.css

```
@@ -972,6 +973,14 @@ kbd {
   }
 }
 
+@media only screen and (max-width: 600px) {
```

Any reason for using 600px here? Given that all doc example should wrapped around 80 chars a line, I think we can use a more fine grained value.
```suggestion
/* 80 chars + the width of the toggle input + 20px of margin */
@media only screen and (max-width: calc(80ch + 142px + 20px)) {
```

---

1. `calc` is not supported in [media query](https://www.w3.org/TR/mediaqueries-4/#media-feature): 
> Properties sometimes accept complex values, e.g., calculations that involve several other values. Media features only accept single values: one keyword, one number, etc.
2. the 80 char lines is a limit in documentation text, not inside `pre` tags that take 100% width

---

> 1. `calc` is not supported in [media query](https://www.w3.org/TR/mediaqueries-4/#media-feature):

That doesn't seem to be correct, according to https://drafts.csswg.org/mediaqueries/#width and `<length>` is a valid value, and IIUC that should include `calc()`.



> 2\. the 80 char lines is a limit in documentation text, not inside `pre` tags that take 100% width

I still think it would be a better value than 600px, that feels quite arbitrary to me.

---

It simply dosent work on chrome

---

Some observations:

- Currently, the longest line to be overlapped with the switch is [`const { createHook, executionAsyncId, triggerAsyncId } = require('node:async_hooks');`](https://nodejs.org/api/async_hooks.html#promise-execution-tracking), 85 characters.
- On my Chrome, overlapping for 80 characters happens on `< 820` and `1024 ~ 1090`.
- On 600px viewport, overlapping starts at ~49 characters.
- Only first two lines might overlap with the switch.

Is it possible/reasonable to add a linter rule in docs for `cjs`/`mjs` blocks, forcing `<=50` or `<=60` for first two lines and maybe `<= 80` for the rest?

Another option is to keep this fix with hardcoded width and eventually somehow redesign the switch, e.g. by attaching it "outside" of code block.